url43a0865bb0_kevinjdolan_darwin_tgz-pJ8-tui_DriverJ8/src/darwInvest/data/gathering/dataSource/GoogleFinanceManual.java

```diff
@@ -0,0 +1,78 @@
+package darwInvest.data.gathering.dataSource;
+
+import java.util.Calendar;
+
+import darwInvest.data.utility.Serializer;
+
+/**
+ * The GoogleFinanceManual reads .csv files generated by Google Finance,
+ * downloaded manually and placed in the folder RawData/GoogleFinanceManual.
+ * It also acts as a utility for automatic methods.
+ * 
+ * @author Andrew Perrault, Kevin Dolan
+ */
+public class GoogleFinanceManual extends CSVReader {
+
+	public GoogleFinanceManual(Serializer serializer) {
+		super("GoogleFinanceManual", serializer);
+	}
+	
+	public int GoogletoMonth(String mon) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 12. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.